.kitchen.yml

```diff
@@ -2,35 +2,33 @@ driver:
   name: vagrant
 
 driver_config:
-  require_chef_omnibus: true
+  require_chef_omnibus: latest
 
 provisioner:
   name: chef_zero
 
 platforms:
-- name: ubuntu-14.04
-  run_list:
-  - recipe[apt]
-
-- name: ubuntu-12.04
-  run_list:
-  - recipe[apt]
-
-- name: ubuntu-10.04
-  run_list:
-  - recipe[apt]
-
-- name: centos-6.5
-
-- name: centos-7.0
+  - name: ubuntu-14.04
+    run_list:
+      - recipe[apt]
+  - name: ubuntu-12.04
+    run_list:
+      - recipe[apt]
+  - name: ubuntu-10.04
+    run_list:
+      - recipe[apt]
+  - name: centos-6.6
```

Comment:
How come you upgraded to 6.6 and 7.1 below? 
Can you confirm this passed kitchen tests?